reference/6/Microsoft.PowerShell.Utility/Invoke-RestMethod.md

```diff
@@ -589,6 +621,23 @@ Position: Named
 Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
+
+```
+### -SkipCertificateCheck
+Skips certificate validation checks.
+
+This includes all validations such as expiration, revocation, trusted root authority, etc.
```

Comment:
It would be good to add a statement like:

```none
This switch is only intended to be used against known hosts using a self-signed certificate for testing purposes.
```

Reply:
Done.